mmv1/templates/terraform/examples/iam_workforce_pool_provider_oidc_upload_key.tf.erb

```diff
@@ -0,0 +1,32 @@
+resource "google_iam_workforce_pool" "pool" {
```

Comment:
This file needs to be referenced from the `examples` block in the yaml configuration for this resources, as described in https://googlecloudplatform.github.io/magic-modules/develop/test/#add-a-create-test

Reply:
fixed